src/language-graphql/printer-graphql.js

```diff
@@ -7,6 +7,18 @@ const { isNextLineEmpty, isNonEmptyArray } = require("../common/util.js");
 const { insertPragma } = require("./pragma.js");
 const { locStart, locEnd } = require("./loc.js");
 
+function hasQueryComment(options, node) {
+  if (
+    options.originalText
+      .slice(locStart(node), locEnd(node))
+      .replace(/\s/g, "")
+      .includes("query{#")
```

Comment:
Nice @Harry-Hopkinson, thanks for yours attempts and courage for fixing actual problem, but imagine if you have named query? Or unnamed/named subscription or mutation or query without `query` keyword, I think you can start by adding failling tests to found a way to fix it 👍